services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -359,10 +370,10 @@ async def inspect_container(
 @containers_router.post(
     "/containers/state:restore",
     summary="Restores the state of the dynamic service",
-    response_model=None,
+    response_class=Response,
```

Comment:
does that make sense? you have a response class but you response with None?

Reply:
@sanderegg  This entrypoint *responds* with a 204 and the handler *returns* ``None``.  This is the most compact way to respond with a 204 in fastapi (and even report it in the OAS).  Notice that the default response in fastapi is Json.

SEE https://github.com/pcrespov/sandbox-python/blob/fastapi-stuff/fastapi-lib/no_content_response.py
NOTE: I would have drop the ``None`` since it is implicit in the 204

Reply:
I will not remove the None, don't want to argue with mypy.